stable/kube-state-metrics/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+name: kube-state-metrics
+description: Install kube-state-metrics to generate and expose cluster-level metrics
+keywords:
+- metric
+- monitoring
+- prometheus
+version: 0.4.1-1
```

Comment:
We want to start/resume versioning charts independently from the application. Let's make this `0.1.0`